README.md

```diff
@@ -13,7 +13,7 @@ Laravel HTMLMin is currently maintained by [Raza Mehdi](https://github.com/srmkl
 
 ## Installation
 
-Laravel HTMLMin requires [PHP](https://php.net) 5.5+. This particular version supports Laravel 5.1, 5.2, 5.3, or 5.4 only.
+Laravel HTMLMin requires [PHP](https://php.net) 5.5+, and supports Laravel 5.1+.
```

Comment:
Not 5.1+, it's 5.1, 5.2, 5.3, 5.4, and 5.5 only.

Reply:
No problem. 👍

I'm curious as to why. Is it for some sort of future-proofing?